### DISCUSSION TITLE
New icons hide

### DISCUSSION THREAD

**Discussion by fafugg** [dwarvesf/hidden#265] [Q&A]
I have a VPN that I open occasionally and want to be always shown (so I know if I am connected to the VPN).
The problem is that every time I open it, the icon gets hidden for some reason. I move it out of the hidden bar arrow every time, but when I quit the program, and then re-open it, it will be hidden again.

Does anyone know how to change that behavior or if there is any file to change the default programs that are hidden?

Thanks

**Comment by qns7** [comment]
I have the same behaviour with some apps and wrote an email to the adress mentioned in the "about"-tab of hidden bar. Perhaps we get a fix...

PS: If you found a solution, please tell me. Perhaps bartender works better? But it's expensive for the simple thing I need it to do...
